app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -100,7 +100,7 @@ $include-html-global-classes: $include-html-classes;
 // $snow        : #F9F9F9;
 // $vapor       : #F6F6F6;
 // $white-smoke : #F5F5F5;
-// $silver      : #EFEFEF;
+ $silver      : #EFEFEF;
```

Comment:
Color `#EFEFEF` should be written as `#efefef`<br>Line should be indented 0 spaces, but was indented 1 spaces